web/Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
```

Comment:
Okay so we make a new source for pypi and this is where we will look for those packages? How did you make this file? just text editing? I haven't seen this before is it for pipenv? it looks like a list of lists of sources, list of packages etc.

Reply:
> Okay so we make a new source for pypi and this is where we will look for those packages?

yes, this allows for private registries as well as the main public one of PyPI.

> How did you make this file? 

It is generated when running `pipenv install <package>` for the first time.

> I haven't seen this before is it for pipenv?

Yes the Pipfile is the pipenv equivalent of a Package.json

>  it looks like a list of lists of sources, list of packages etc.

The [[source]] is the sources, later on is the packages with fast api and uvicorn